ecc_go/chaincode/enclave_go/skvs_stub.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package enclave_go
+
+import (
+	"crypto/sha256"
+	"fmt"
+
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+	"github.com/hyperledger/fabric-private-chaincode/internal/protos"
+	"github.com/hyperledger/fabric/protoutil"
+	"github.com/pkg/errors"
+	"google.golang.org/protobuf/proto"
+)
+
+type skvsStub struct {
+	*EnclaveStub
+}
+
+func NewSkvsStub(cc shim.Chaincode) *skvsStub {
+	enclaveStub := NewEnclaveStub(cc)
+	return &skvsStub{enclaveStub}
+}
+
+func (e *skvsStub) ChaincodeInvoke(stub shim.ChaincodeStubInterface, chaincodeRequestMessageBytes []byte) ([]byte, error) {
```

Comment:
I have the feeling that we should remove this wrapper and instead just inject a provider function for the stub that we can set externally.

Reply:
no sure about this one, might need to ask more insight